acceptance/bundle/validate/include_locations/output.dev.json

```diff
@@ -0,0 +1,100 @@
+{
+  "bundle": [
+    {
+      "column": 3,
+      "file": "databricks.yml",
+      "line": 2
+    }
+  ],
+  "bundle.name": [
+    {
+      "column": 9,
+      "file": "databricks.yml",
+      "line": 2
+    }
+  ],
+  "include": [
+    {
+      "column": 3,
+      "file": "databricks.yml",
+      "line": 5
+    }
+  ],
+  "include[0]": [
+    {
+      "column": 5,
+      "file": "databricks.yml",
+      "line": 5
+    }
+  ],
+  "include[1]": [
+    {
+      "column": 5,
+      "file": "databricks.yml",
+      "line": 6
+    }
+  ],
+  "resources": [
+    {
+      "column": 3,
+      "file": "job.yml",
+      "line": 2
+    }
+  ],
+  "resources.jobs": [
+    {
+      "column": 5,
+      "file": "job.yml",
+      "line": 3
+    }
+  ],
+  "resources.jobs.my_job": [
```

Comment:
I wonder if we should encode this like 

`"resources['jobs']['my_job']"` 

to cover cases where the keys contain spaces or dots

Reply:
Good point.

This can happen only for map keys (e.g. `spark_conf`). The rest of the configuration is fully typed and only includes alphanumeric fields. Since we don't care about map entries perse (they are included), we could ignore this.

Reply:
Make sense. Then we have to make sure that we don't emit source maps for entries that are not alpha-numeric